swediversity/src/views/CityInfo/EatDrink.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+    <div>
+        <div class="bestOfBest">
+            <div class="title-section main-font">
+                <h1>명예의 전당</h1>
+            </div>
+            <div class="content-section">
+                <div class="row">
+                    <div class="col">
+                        <div class="box">
+                            <div class="box-content">
+                                <div class="box-1"></div>
+                            </div>
+                            <div class="box-title-story">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col">
+                        <div class="box" style="align-self: flex-start;">
+                            <div class="box-content">
+                                <div class="box-1"></div>
+                            </div>
+                            <div class="box-title-story">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col">
+                        <div class="box" style="align-self: flex-end;">
+                            <div class="box-content">
+                                <div class="box-1"></div>
+                            </div>
+                            <div class="box-title-story">
+                            </div>
+                        </div>
+                    </div>
+                </div>
```

Comment:
## Problem
The same code is being repeated too many times making the code less DRY.
This is a bad habit but I did this in this way because there wasn't a way to adjust the boxes in different positions.